Process.c

```diff
@@ -36,6 +36,9 @@ in the source distribution for its full text.
 #endif
 
 
+/* Used to identify kernel threads in Comm and Exe columns */
+static const char *const kthreadID = "KTHREAD";
```

Comment:
Most of the new code aligns the pointer specifier to the type on the left.
While this is generally also my preference (e.g. to avoid `char* a, b;`), most of the existing code uses right-/name-alignment and the style-guide also mentions `astyle ... -k1` (= `--align-pointer=type`).

Should we adjust the style guide to allow both types of alignment (with or without a preference)?

Reply:
This has been taken 1:1 from the current dev branch, just moved over from `linux/LinuxProcess.c` …

I'm indifferent on that one.

Reply:
Added to meeting agenda